docs/_ug/commandSummary/itemCommands/find.md

```diff
@@ -1,3 +1,2 @@
 <!-- markdownlint-disable-file first-line-h1 -->
-find ITEM_NAME [ITEM_NAME]
-Note: You can provide multiple keywords to find an item. For example, `find potato chips`.
+`find KEYWORD [KEYWORD]...`
```

Comment:
Hmm we might have to rewrite this...

Reply:
May I know what you mean by rewrite

Reply:
Cause it's not immediately clear how the find command works. Are the keywords counted as two separate independent searches, or what. I guess we can leave this in for now but just open an issue to look into the phrasing again.

Reply:
Have created an issue